$MirOS: ports/audio/libogg/patches/patch-doc_Makefile_am,v 1.1 2005/11/03 19:23:35 bsiegert Exp $
--- doc/Makefile.am.orig	Mon Nov 28 04:25:44 2005
+++ doc/Makefile.am	Wed Dec 28 15:51:59 2005
@@ -2,7 +2,7 @@
 
 SUBDIRS = libogg
 
-docdir = $(datadir)/doc/$(PACKAGE)-$(VERSION)
+docdir = $(datadir)/doc/$(PACKAGE)
 
 doc_DATA = framing.html index.html oggstream.html ogg-multiplex.html \
 	stream.png vorbisword2.png white-ogg.png white-xifish.png \
